### PR TITLE
3 notebooks: add deployment and inference to multilabel, multiclass and NER

### PR DESCRIPTION
**Pull request by MelonXiaoxiao**
multiclass, multilabel and NER model.

# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- add model deployment and inference sections to NLP Multilabel, Multiclass and NER for v2

fixes #

### CODE REVIEW

**Comment by MelonXiaoxiao** [comment]
> The way to get the best run and the best model seems very complicated. Can you use something like this instead?
> 
> ```
> if returned_sweep_job.status == "Completed":
> 
>     # First let us get the run which gave us the best result
>     best_run = returned_sweep_job.properties["best_child_run_id"]
> 
>     # lets get the model from this run
>     model = Model(
>         # the script stores the model as "outputs"
>         path="azureml://jobs/{}/outputs/artifacts/paths/outputs/".format(best_run),
>         name="run-model-example",
>         description="Model created from run.",
>         type="custom_model",
>     )
> 
> else:
>     print(
>         "Sweep job status: {}. Please wait until it completes".format(
>             returned_sweep_job.status
>         )
>     )
> ```
Adjusted accordingly; the model artifacts are being extracted without the need of local download.

File: sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb [hunk 1]
```diff
@@ -1,517 +1,900 @@
...
+        "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# 1. Connect to Azure Machine Learning Workspace\n",
+        "\n",
+        "The [workspace](https://docs.microsoft.com/en-us/azure/machine-learning/concept-workspace) is the top-level resource for Azure Machine Learning, providing a centralized place to work with all the artifacts you create when you use Azure Machine Learning. In this section we will connect to the workspace in which the job will be run.\n",
```
Comment (harneetvirk) [inline sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb:24]: Let's not include locale information in the url. Please remove en-us

File: sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb [hunk 2]
```diff
@@ -1,517 +1,900 @@
...
+        "from pprint import pprint"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 1.2. Configure workspace details and get a handle to the workspace\n",
+        "\n",
+        "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
```
Comment (harneetvirk) [inline sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb:54]: same, locale information in the link

Comment (MelonXiaoxiao) [reply sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb:54]: en-us removed for all three notebooks

File: sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb (hunk 1 above)
Comment (MelonXiaoxiao) [reply sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-text-classification-multiclass-task-sentiment-mlflow.ipynb]: en-us removed for all three notebooks in weblinks

**Review by harneetvirk — APPROVED** [review]

**Comment by balapv** [comment]
@harneetvirk can you take a look at this one?

**Review by balapv — APPROVED** [review]

**Pull request closed without merging**